.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,9 +21,9 @@ If your code changes how something works on the device (i.e., it affects the com
 
 Further, if you've changed the blocks language or another user-facing designer/blocks API (added a SimpleProperty, etc.):
 
-- [ ] I have updated the corresponding version number in YaVersion.java
-- [ ] I have updated the corresponding upgrader in YoungAndroidFormUpgrader.java (components only)
-- [ ] I have updated the corresponding entries in versioning.js
+- [ ] I have updated the corresponding version number in appinventor/components/common/YaVersion.java
+- [ ] I have updated the corresponding upgrader in appinventor/client/youngandroid/YoungAndroidFormUpgrader.java (components only)
+- [ ] I have updated the corresponding entries in appinventor/blocklyeditor/src/versioning.js
```

Comment:
I wonder if we should include `...` in the paths to indicate that they are not complete.

Reply:
> I wonder if we should include `...` in the paths to indicate that they are not complete.

That makes sense.